primitives/xcm/src/asset_id_conversions.rs

```diff
@@ -59,6 +59,10 @@ pub trait AssetTypeGetter<AssetId, AssetType> {
 
 	// Get assetId from assetType
 	fn get_asset_id(asset_type: AssetType) -> Option<AssetId>;
+
+	// Set assetId and assetType
+	#[cfg(feature = "runtime-benchmarks")]
+	fn set_asset_type_asset_id(asset_type: AssetType, asset_id: AssetId);
```

Comment:
Simalarly to my other suggestion, you could prefix this with `testing_` or similar to avoid any confusion over it.

Reply:
While I agree with the previous suggestion, I find the pattern on this one very common (e.g., https://github.com/paritytech/cumulus/blob/3fb9c8a7beb8fb0fc31ea3335f8483d8086f60c8/pallets/parachain-system/src/lib.rs#L1113).

Reply:
Yeah, I've seen the pattern a lot. I agree that following convention is a good thing.